src/components/vue-form-builder.vue

```diff
@@ -358,6 +361,12 @@ export default {
     isCurrentPageEmpty() {
       return this.config[this.currentPage].items.length === 0;
     },
+    showToolbar() {
+      if (!this.screenType) {
+        return;
```

Comment:
This reads backwards to me. If `showToolbar` is `false`, the toolbar is shown. I'd assume that `showToolbar === true` would mean the toolbar should be shown.

When you don't specify a return value for a function, it implicitly returns `undefined`, however, I think in this case it should always return a boolean. So the above line could be `return false;`.

I'd change the whole thing to something like:

```
return this.screenType === formTypes.display;
```

And ensure that the `screenType` prop gets a default value of `formTypes.display` if it's not set.